Mirror/Runtime/Messages.cs

```diff
@@ -411,6 +411,43 @@ public override void Serialize(NetworkWriter writer)
         }
     }
 
+    // A client sends this message to the server 
+    // to calculate RTT and synchronize time
+    class NetworkPingMessage : MessageBase
```

Comment:
In Message.cs we have 'general typed messages' at the top, like StringMessage and IntegerMessage, maybe we should add a DoubleMessage and inherit from it? Might be useful later too.